test/example.test.js

```diff
@@ -1,16 +1,71 @@
-// IMPORT MODULES under test here:
-// import example from '../src/example.js';
+import { albumsArray , cart } from '../Shopping-Cart/albumsArray.js';
```

Comment:
It looks like you moved some files around. Make sure you change the paths accordingly or you'll break your tests.